globals.py

```diff
@@ -1,7 +1,7 @@
 #! /usr/bin/env python
 
 from datetime import datetime
-import threading, sys, os, Queue#, thread
+import threading, sys, os, queue#, thread
```

Comment:
This won't work on python 2, we should support both. We could use six (https://pythonhosted.org/six/) to do this, however that would be a new external dependency, which we don't tend to place on users (though I'm open to changing things up to do this).

Alternatively, could try/except ImportError to do the right thing.